perma_web/perma/migrations/0006_add_internetarchive_status.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+def update_upload_to_ia_field(apps, schema_editor):
+    Link = apps.get_model('perma', 'Link')
```

Comment:
These will deploy a *lot* faster on prod if we do them as something like, `Link.objects.filter(uploaded_to_internet_archive=True).update(internet_archive_upload_status='completed')`